quinn/src/endpoint.rs

```diff
@@ -137,7 +137,13 @@ impl Future for Driver {
                 break;
             }
         }
-        Ok(Async::NotReady)
+        Ok(
+            if endpoint.unreferenced && endpoint.connections.is_empty() {
+                Async::Ready(())
+            } else {
+                Async::NotReady
+            },
```

Comment:
The comma here seems unidiomatic?

Reply:
rustfmt insists, unfortunately. Might be worth opening an issue there?

Reply:
Ugh.